plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TestCheckpointEntryIterator.java

```diff
@@ -600,10 +699,14 @@ private CheckpointEntryIterator createCheckpointEntryIterator(
                 metadataEntry,
                 protocolEntry,
                 new FileFormatDataSourceStats(),
-                new ParquetReaderConfig().toParquetReaderOptions(),
+                new ParquetReaderConfig()
+                        .setMaxBufferSize(DataSize.ofBytes(500))
```

Comment:
These changes were necessary to be able to showcase the effectiveness of the `stats_parsed` projection functionality.